go.mod

```diff
@@ -168,6 +168,99 @@ require (
 	sigs.k8s.io/yaml v1.2.0 // indirect
 )
 
+require (
```

Comment:
Bit confused why all these pkgs are appearing.. Can you do a `go clean -modcache` and `go mod tidy`? Also make sure to do this before go mod tidy `export GOPROXY=`. Otherwise, sometimes it tries to pick libraries from artifactory